index.js

```diff
@@ -0,0 +1,42 @@
+class Task {
+  constructor(id, title, description) {
+    this.id = id;
+    this.title = title;
+    this.description = description;
+  }
+}
+
+class TaskList {
+  constructor() {
+    this.tasks = [];
+  }
+  generateId() {
+    return `${Date.now()}-${Math.floor(Math.random() * 10000)}`;
+  }
```

Comment:
No need to create function here, you can use that random num inside the `Task` class immediately (`this.id =`${Date.now()}-${Math.floor(Math.random() * 10000)}``)